staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2842,6 +2940,40 @@ func (i *IngressClassDescriber) Describe(namespace, name string, describerSettin
 	return "", err
 }
 
+func (i *IngressClassDescriber) tryCastNetworkingV1IngressClass(object runtime.Object) (*networkingv1.IngressClass, error) {
```

Comment:
Same comment as before, just use `networking/v1.IngressClass`. It's similarly [deprecated since 1.19](https://github.com/kubernetes/kubernetes/blob/790393ae92e97262827d4f1fba24e8ae65bbada0/staging/src/k8s.io/api/networking/v1beta1/types.go#L335).